packages/server/src/threads/automation.ts

```diff
@@ -465,3 +472,15 @@ export function execute(input: AutomationEvent, callback: WorkerCallback) {
     }
   })
 }
+
+export const removeStalled = (input: AutomationEvent) => {
```

Comment:
This isn't thread related but this is the easiest way to integrate with the existing cron disabling. We should consider moving the orchestrator / automation specific logic out of the thread class in future into the top level automations package